assets/src/edit-story/app/media/useUploadMedia/useUploadMedia.js

```diff
@@ -104,24 +104,37 @@ function useUploadMedia({ media, pagingNum, mediaType, fetchMedia, setMedia }) {
       }
 
       try {
-        const uploadingFiles = await Promise.all(
+        const uploadedFiles = await Promise.all(
           localFiles.map(async (localFile) => ({
             ...localFile,
-            fileUploaded: await uploadFile(localFile.file),
+            fileUploaded: getResourceFromAttachment(
+              await uploadFile(localFile.file)
+            ),
           }))
         );
 
         setIsUploading(false);
 
         if (onUploadedFile) {
-          uploadingFiles.forEach(({ element, fileUploaded }) => {
+          uploadedFiles.forEach(({ element, fileUploaded }) => {
             onUploadedFile({
-              resource: getResourceFromAttachment(fileUploaded),
+              resource: fileUploaded,
               element,
             });
           });
         }
-        fetchMedia({ pagingNum, mediaType }, setMedia);
+
+        const uploadedFilesMap = Map(
```

Comment:
```suggestion
        const uploadedFilesMap = new Map(
```

Reply:
![Screenshot 2020-05-21 at 15 41 22](https://user-images.githubusercontent.com/237508/82571555-07a8b100-9b7b-11ea-8551-7358663754f2.png)
Fixes this issue, I was seeing.